metadata/ComputerVision__9__metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+  "changeLog": "Release v2022.10.10",
+  "cpu": 0,
+  "description": "Backend server for the Computer Vision solution that detects User Interface Elements from a provided Application screenshot",
+  "displayName": "ComputerVision",
+  "gpu": 1,
+  "inputDescription": "Please use the Computer Vision Activities included in the UIAutomation package. For more information please visit the official documentation. https://docs.uipath.com/activities/other/latest/ui-automation/computer-vision-activities ",
+  "inputType": "JSON",
+  "memory": 0,
+  "mlPackageLanguage": "PYTHON36_CV",
+  "name": "ComputerVision",
+  "outputDescription": "Please use the Computer Vision Activities included in the UIAutomation package. For more information please visit the official documentation.",
```

Comment:
Same on this line please. Please try to read all of the lines of this file, compare them with `23.10` and understand why they are different for each line that is different, as much as reasonably possible.

Reply:
Done.